packages/app/src/ErrorFallback.tsx

```diff
@@ -10,7 +10,27 @@ interface Props extends FallbackProps {
 function ErrorFallback(props: Props) {
   const { className = '', error, resetErrorBoundary } = props;
 
-  if (error.cause || error.cause instanceof Error) {
+  if (error.message === CANCELLED_ERROR_MSG) {
+    return (
+      <p className={`${styles.error} ${className}`}>
+        {CANCELLED_ERROR_MSG}
+        <span>–</span>
+        <button
+          className={styles.retryBtn}
+          type="button"
+          onClick={() => resetErrorBoundary()}
+        >
+          Retry?
+        </button>
+      </p>
+    );
+  }
+
+  if (
+    error.cause &&
+    error.cause instanceof Error &&
+    error.message !== error.cause.message
```

Comment:
No longer rendering an accordion if the cause error message is the same as the error message.